MNE/fs/surface.cpp

```diff
@@ -200,6 +200,8 @@ bool Surface::read(const QString &subject_id, qint32 hemi, const QString &surf,
 
     QString p_sFile = QString("%1/%2/surf/%3.%4").arg(subjects_dir).arg(subject_id).arg(hemi == 0 ? "lh" : "rh").arg(surf);
 
+    qDebug()<<p_sFile;
```

Comment:
If this should stay permanent please consider printf instead of Debug. and add something pore to the command line output. Like: "Try to read: ...."